app/assets/stylesheets/unsubscriptions.css.scss

```diff
@@ -0,0 +1,14 @@
+// Place all the styles related to the Unsubscriptions controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.unsubscriptions-controller {
+  width: 70%;
+  margin: 0 15%;
+}
+
+.email-categories {
+  padding: 10px;
+  // border: solid 1px #444;
+  margin: 5px 5px;
+}
```

Comment:
you have a set up your text editor to include a new blank line at the end the file when you saving.